<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T02_Dialog</class>
 <widget class="QDialog" name="T02_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>T02_Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox_Bold">
        <property name="text">
         <string>Bold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Italic">
        <property name="text">
         <string>Italic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Underline">
        <property name="text">
         <string>Underline</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_Blue">
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_Red">
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_Green">
        <property name="text">
         <string>Green</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="plainText">
      <string>剑光如我，斩尽芜杂！</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Exit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>T02_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Exit</sender>
   <signal>clicked()</signal>
   <receiver>T02_Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>T02_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
